### PR TITLE
KPV-5975 Create an url shortener

### PR DESCRIPTION
**Pull request by PedroR-dev**
Add new URL mapping for all campaigns types.
URL format is domainUrl/s/{campaignId}.
Works for logged and no-logged users

### CODE REVIEW

**Comment by PedroR-dev** [comment]
Incognito mode: image

Logged user: image

**Comment by PedroR-dev** [comment]
I think cache is not necessary for this mapping because the url to which it redirects already has the cache implemented. Am I right?

File: grails-app/controllers/kuorum/politician/CampaignController.groovy [hunk 1]
```diff
@@ -140,6 +140,15 @@ class CampaignController {
         }
     }
 
+    def showShort(){
+        print("Using shortener, redirecting...")
```
Comment (iduetxe) [inline grails-app/controllers/kuorum/politician/CampaignController.groovy]: Remove this print

File: grails-app/controllers/kuorum/politician/CampaignController.groovy [hunk 2]
```diff
@@ -140,6 +140,15 @@ class CampaignController {
...
+        int httpCode = HttpServletResponse.SC_MOVED_PERMANENTLY
+        String campaignId = params.campaignId;
+        response.setStatus(httpCode)
+        CampaignRSDTO campaignRSDTO = campaignService.find(String.valueOf(WebConstants.FAKE_LANDING_ALIAS_USER), Long.parseLong(campaignId))
```
Comment (iduetxe) [inline grails-app/controllers/kuorum/politician/CampaignController.groovy:148]: Insetead of FAKE_LANDING_ALIAS_USER use something that you understand that is a short url ÑAPA

**Review by iduetxe — APPROVED** [review]

**Pull request merged**